### DISCUSSION TITLE
Ghostty displays slightly incorrect palette colors

### DISCUSSION THREAD

**Discussion by edrozenberg** [ghostty-org/ghostty#3902] [Q&A] [answered]
Ghostty Version 1.0.0
Build 8469
Commit https://github.com/ghostty-org/ghostty/commit/4b4d4062dfed7b37424c7210d1230242c709e990
(MacOS)

The colors that Ghostty actually displays are slightly shifted from palette codes defined in `config`.

For ex:
In `config` file:
`palette = 04=#3840b2`

Ghostty actually displays:
`#3740b2`
(according to my Mac color picker utility Sip)

**Comment by edrozenberg** [comment]
This might be related to an existing issue which says:
> ...we don't do proper gamma correction from our linear RGB values that are used everywhere else (bg, fg, palette, SGR styles, etc.).
...
> The result is that all specified color is a little bit off compared to native GUI toolkits 

Issue: Gamma Correction https://github.com/ghostty-org/ghostty/issues/2125

**Comment by pluiedev** [comment, marked as answer]
Resolved on macOS with https://github.com/ghostty-org/ghostty/pull/4913
